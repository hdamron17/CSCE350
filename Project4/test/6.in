12345678
4567890
88888888888
1234567890123456789012345678901234567890
2223
