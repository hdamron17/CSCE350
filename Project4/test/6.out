3
bcdefghi
bxefghi
mdefghi
1
efghija
1
iiiiiiiiiii
81
5
cccd
ccx
cwd
wcd
wx
